### DISCUSSION TITLE
Left align tooltip text when it spans for more than one line

### DISCUSSION THREAD

**Discussion by muthubentley** [iTwin/iTwinUI#1103] [Q&A] [answered]
Referred to below URL and implemented tooltip. 
https://itwin.github.io/iTwinUI/react/?path=/story/core-tooltip--top
![image](https://user-images.githubusercontent.com/107229931/222108033-546c3406-7d43-47b0-97ae-cd5fc4cc8e60.png)

When the tooltip text is large and spans to two lines, it become center aligned.
Partner wants it to be left aligned. Can you please let me know how to make them left aligned.

**Comment by mayank99** [comment, marked as answer]
Hey @muthubentley, you can pass a custom `className` and apply your own styles.

```jsx
<Tooltip className='my-tooltip'>...</Tooltip>
```
```css
.my-tooltip {
  text-align: left;
}
```
https://codesandbox.io/s/dark-snowflake-4dq24s?file=/src/App.tsx

**Reply by muthubentley** [reply]
Thanks @mayank99  . Will refer to this
